pkg/utils/dataset.go

```diff
@@ -71,6 +72,25 @@ func GetAccessModesOfDataset(client client.Client, name, namespace string) (acce
 
 }
 
+func GetPVCStorageCapacityOfDataset(client client.Client, name, namespace string) (storageCapacity resource.Quantity, err error) {
+	dataset, err := GetDataset(client, name, namespace)
+	if err != nil {
+		return storageCapacity, err
+	}
+	annotations := dataset.GetObjectMeta().GetAnnotations()
+	size := annotations["pvc.fluid.io/resources.requests.storage"]
+	if size == "" {
+		storageCapacity = resource.MustParse("100Pi")
```

Comment:
I suggest using const defaultStorageCapacity as a constant, so changing the default becomes more manageable.

```golang
// Default storage capacity constant
const defaultStorageCapacity = "100Pi"
```